version.json

```diff
@@ -1,3 +1,3 @@
 {
-	"version": "v0.1.0"
+	"version": "v0.1.1"
```

Comment:
This probably needs to be v0.2.0, as you're dropping a dependency.

Reply:
The CI disagree. 🙂 

https://github.com/multiformats/go-multihash/runs/6705660054?check_suite_focus=true

Reply:
It does not:
```
`gorelease` says:
  ```
  # summary
  Suggested version: v0.2.0
  ```
```